docs/examples/DocsExamples/Streams/StreamTcpDocTests.cs

```diff
@@ -0,0 +1,137 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Akka.Streams;
+using Akka.Streams.Dsl;
+using Akka.TestKit.Xunit2;
+using Xunit;
+using Xunit.Abstractions;
+using Akka.Actor;
+using Akka.IO;
+using Tcp = Akka.Streams.Dsl.Tcp;
+using TcpExt = Akka.Streams.Dsl.TcpExt;
+using Akka.Util.Internal;
+
+namespace DocsExamples.Streams
+{
+    public class StreamTcpDocTests : TestKit
+    {
+        private ActorMaterializer Materializer { get; }
+
+        public StreamTcpDocTests(ITestOutputHelper output)
+            : base("", output)
+        {
+            Materializer = Sys.Materializer();
+        }
+
+        [Fact]
+        public async Task Simple_server_connection_must_bind_and_unbind()
+        {
+
+            #region echo-server-simple-bind
+            var tcp = new TcpExt(Sys.AsInstanceOf<ExtendedActorSystem>());
```

Comment:
This is ugly API (it should be hidden IMHO). `Sys.TcpStream()` looks a lot better.

Reply:
Ok :)